### ISSUE TITLE
What we build?!

### ISSUE THREAD

**Issue by rkernan** [bhand/learn_git#1]
Ideas?

I'm thinking some simple REST but no idea what. Thoughts?

**Comment by bhand** [comment]
I've always kind of wanted to build a chat application. I think it's a nice project for golang especially. What do you think? These applications are already a dime a dozen but I say screw it.

**Comment by rkernan** [comment]
An XMPP application could be interesting but I'm not sure how much back end work there is there. We could also use Web Sockets which may be more interesting from an implementation perspective. It depends on our goal.

**Comment by bhand** [comment]
Yeah, I had WebSockets in mind when I thought of this. Just something new.

**Comment by rkernan** [comment]
WebSockets it is, then.

**Comment by bhand** [comment]
Looks like this is the go implementation of WebSockets: https://github.com/gorilla/websocket
